action.yml

```diff
@@ -69,6 +69,13 @@ inputs:
     description: 'A prefix to apply to the version number, which reflects in the tag and GitHub release names'
     required: true
     default: 'v'
+  monorepo-package:
+    description: 'A specific monorepo package to be deployed. The package should have the full path from the root of the project E.g. `packages/react-app`'
```

Comment:
Yep this was a mistake. At first I thought it would be a good idea for them to be defined together but that quickly proved to be a mistake and I forgot to update the docs later.